tests/test_admin_dashboard_schema_compat.py

```
@@ -0,0 +1,82 @@
+import sqlite3
+import tempfile
+from contextlib import contextmanager
+
+from app.config.settings import settings
+
+
+def test_admin_dashboard_works_with_legacy_user_schema(test_client, monkeypatch):
+    monkeypatch.setattr(settings, "ADMIN_USER", "admin")
+    monkeypatch.setattr(settings, "ADMIN_PASS", "secret123")
+
+    fd, db_path = tempfile.mkstemp(suffix=".db")
```

Ruff (0.15.10)

[warning] Unpacked variable `fd` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)